Backend/TicTacToe/TicTacToe/Controllers/UserChoiceController.cs

```diff
@@ -4,22 +4,21 @@
 
 namespace TicTacToe.Controllers;
 
-
 [ApiController]
 [Route("[Controller]")]
```

Comment:
suggestion: keep consistent naming conventions

```suggestion
[Route("[controller]")]
```